erpnext/hr/doctype/holiday_list/holiday_list.py

```diff
@@ -90,7 +90,41 @@ def is_holiday(holiday_list, date=today()):
 	"""Returns true if the given date is a holiday in the given holiday list
 	"""
 	if holiday_list:
-		return bool(frappe.get_all('Holiday List',
-			dict(name=holiday_list, holiday_date=date)))
+		return bool(frappe.get_all('Holiday List',dict(name=holiday_list, holiday_date=date)))
 	else:
 		return False
+
+def send_holiday_notification():
+	# holidays is a list of holidays which fall in 7 days from today
+	holiday_lists = frappe.get_all('Holiday List', fields = ["name", "send_reminders_to","notification_message"])
```

Comment:
@hrwX Should we add a "active" checkbox in Holiday List to avoid unnecessary looping over old holiday lists?

Reply:
@sahil28297 There can be multiple lists active at a time for example, On our HR system, we have 3 different holiday lists that are for the current year, You can check it out here: https://bloomstack.com/desk#List/Holiday%20List/List

Reply:
yes, we can check that checkbox for multiple holiday list.

Reply:
@sahil28297 That sure can be done, but we cannot eliminate the looping
@hrwX Your views?

Reply:
@sahil28297 Made a commit, which gets only those holiday_lists where today's date falls between the holiday start date and holiday end date